src/pages/ProjectsView.jsx

```diff
@@ -14,17 +12,16 @@ export default function ProjectsView() {
 
   return (
     <div className="projects-view-container">
-      <div className="projects-view-body">
-        <div className="projects-view-copy-container">
-          <h1>Projects</h1>
-          <FeatureContainer image={<ProjectsSVG />}>
-            <FeatureCopy>
-              Contribute to one of our member-owned projects! Can’t find what you are looking for? Bring in a new project idea or existing project!
-            </FeatureCopy>
-            <Button handleClick={() => window.open(SocialMediaConstants.DISCORD_LINK, "_blank")}>Join our Discord</Button>
-          </FeatureContainer>
+          <PageHeader title='Projects' img={<ProjectsSVG />}>
+              <p>
+                Contribute to one of our member-owned projects! Can’t find what you are looking for? Bring in a new project idea or existing project!
+              </p>
+          </PageHeader>
+      <div className="cards-container">
```

Comment:
could you name this class Name projects-card-container? Just so we have it be more specific not to have it globally impacted at any point